play/binary-explorer/learn/introduction.html

```diff
@@ -49,7 +49,7 @@ <h1 class="text-center mt-2">
 			</h1>
 			
 			<p class="text-right">
-				<button type="button" class="btn btn-success" onclick="location.href='binary-numbers.html'">Next</button>
```

Comment:
With Bootstrap you can actually just add the btn class to an ```<a>``` element to make it look like a button to have to avoid using onclick JavaScript code like this.

Reply:
Ah, I see. Thank you!